lib/sign.js

```diff
@@ -24,6 +26,11 @@ function sign (program, options, config) {
     AMOClient: DefaultAMOClient,
```

Comment:
`config.systemProcess` is no longer used in this function so you need to set the default for it in `signCmd` instead. Actually, the `jpm sign` command won't function without this change.

At the top of the `signCmd` just do something similar:
````
config = _.assign({
  systemProcess: process,
}, config);
````
This allows the tests to replace `systemProcess` but the real command to use node's magic `process` variable.